package.json

```diff
@@ -14,33 +16,43 @@
 		}
 	},
 	"scripts": {
-		"prebuild": "npm run check-engines && npm run build:css",
-		"prewatch": "npm run check-engines",
-		"watch": "concurrently \"webpack --watch --config webpack.dev.js\" \"npm-watch build:css\"",
 		"build": "cross-env NODE_ENV=production webpack --config webpack.prod.js",
-		"watch-poll": "webpack --watch --watch-poll --config webpack.dev.js",
-		"lint": "wp-scripts lint-js .",
-		"lint:fix": "wp-scripts lint-js . --fix",
+		"build:css": "node ./bin/css-builder/css-builder.js",
 		"check-engines": "wp-scripts check-engines",
-		"ci:test": "npm run test-unit:coverage-ci",
-		"ci:lint": "npm run lint",
 		"ci:build": "npm run build",
+		"ci:lint": "npm run lint",
+		"ci:test": "npm run test-unit:coverage-ci",
+		"lc": "jsgl --local ./",
+		"lint": "wp-scripts lint-js .",
+		"lint:fix": "wp-scripts lint-js . --fix",
+		"lint-js": "wp-scripts lint-js",
+		"lint-js:fix": "npm run lint-js -- --fix",
+		"lint-php": "composer run-script lint",
+		"lint-php:fix": "composer run-script lint-fix",
+		"lint-php:skip-warnings": "composer run-script lint:skip-warnings",
+		"prebuild": "npm run check-engines && npm run build:css",
+		"prewatch": "npm run check-engines",
 		"test": "npm run lint && npm run test-unit",
 		"test-php": "cd tests && phpunit",
 		"test-php:coverage-ci": "npm run test-php -- --coverage-clover=coverage.xml && codecov",
-		"lint-php": "composer run-script lint",
-		"lint-php:skip-warnings": "composer run-script lint:skip-warnings",
-		"lint-php:fix": "composer run-script lint-fix",
 		"test-unit": "wp-scripts test-unit-js --config tests/javascript-config/unit/jest.config.json",
-		"test-unit:update": "npm run test-unit -- --updateSnapshot",
-		"test-unit:watch": "npm run test-unit -- --watch",
 		"test-unit:coverage": "npm run test-unit -- --coverage",
 		"test-unit:coverage-ci": "npm run test-unit -- --coverage --maxWorkers 1 && codecov",
-		"lc": "jsgl --local ./",
-		"build:css": "node ./bin/css-builder/css-builder.js"
+		"test-unit:update": "npm run test-unit -- --updateSnapshot",
+		"test-unit:watch": "npm run test-unit -- --watch",
+		"watch": "concurrently \"webpack --watch --config webpack.dev.js\" \"npm-watch build:css\"",
```

Comment:
hey I've been thinking it would be handy to have a command for running a dev build but without the watch flag added.
ie: something like:

```js
		"dev": "concurrently \"webpack --config webpack.dev.js\" \"npm-watch build:css\"",
```

is there already a convention for naming a command like that other than "dev" ?

If so, we can use that...
although imho it would make more sense to have:

- `npm run dev`
- `npm run prod` 

rather than

- `npm run watch`
- `npm run build`

and for the watch and polling dev builds we could have:

- `npm run dev-w` for `webpack --watch --config webpack.dev.js`
- `npm run dev-p` for `webpack --watch --watch-poll --config webpack.dev.js`

what do you think?
I have no experience with these things outside of this project, so if my suggestions are comical, then plz feel free to laugh! I can take it !

Bottom line is that we should follow standard conventions as much as possible for the benefit of all

Reply:
In context of gatsby projects I've seen `dev` being used over the longer `develop`, so I am fine with `dev`, rather than `watch` or `develop`.
I've noticed `build` being more often used than `prod`, but don't want to debate over this, but rather find an agreement and move forward since this is a minor thing.